packages/push/examples/simple_regression/main.rs

```diff
@@ -143,7 +143,7 @@ fn main() -> Result<()> {
 
     let num_test_cases = 10;
 
-    let selector = Weighted::new(Best, 1)
+    let selector = DynWeighted::new(Best, 1)
```

Comment:
Question: Do we want `DynWeighted` or just `Weighted` in the examples?

Reply:
We'll address this in #263.